OpenOversight/app/utils.py

```diff
@@ -473,6 +474,32 @@ def create_description(self, form):
         date_updated=datetime.datetime.now())
 
 
+def conditionally_rotate_image(image):
+    try:
+        # NOTE: Pillow >= 6.0 provides `ImageOps.exif_transpose` which acts like the below code
```

Comment:
Any reason not to bump the Pillow dependency version instead?

Reply:
I’ll try that, too. I was a bit hesitant to mess with dependencies.